apps/super-app/src/services/DetailServices.ts

```diff
@@ -6,7 +6,7 @@ import { Support } from './SupportServices';
 
 // Data shared by every Support detail page.
 
-export interface ServiceCardAbout extends Support {
+export interface ServiceCardDetail extends Support {
```

Comment:
This renaming from `ServiceCardAbout` to `ServiceCardDetail` looks like a leftover from the previous PR iteration. I'd prefer to undo this, but I suppose the name makes more sense, so we can keep it if we want.

Reply:
Oh no, this one is deliberate as I also feel this name makes more sense :)

Reply:
Sure. In the future, it would be helpful to call out intentional changes in the PR message or in a comment.